### ISSUE TITLE
Slim batch image prediciton?

### ISSUE THREAD

**Issue by jameschao2000** [tensorflow/tensorflow#16796]
I just finished training a model by following train_image_classifier.py

CUDA_VISIBLE_DEVICE=0,1 python train_image_classifier.py --train_dir=train_logs --dataset_dir=../train --num_samples=15500 --num_classes=4 --labels_to_names_path=../labels.txt --model_name=inception_resnet_v2 --checkpoint_path=../checkpoints/inception_resnet_v2_2016_08_30.ckpt --checkpoint_exclude_scopes=InceptionResnetV2/Logits,InceptionResnetV2/AuxLogits --num_clones=2 --num_preprocessing_threads=8 --max_number_of_steps=100000 --batch_size=32 --learning_rate=0.0001 --learning_rate_decay_type=fixed --save_interval_secs=60 --save_summaries_secs=60 --log_every_n_steps=10 --optimizer=rmsprop --weight_decay=0.00004

and evaluate the model by using eval_image_classifier.py. 

CUDA_VISIBLE_DEVICE=0,1 python eval_image_classifier.py --checkpoint_path=train_logs --eval_dir=eval_logs --dataset_dir=../val --num_samples=797 --num_classes=4 --model_name=inception_resnet_v2

Everything seems great. But, how to test image classifier unfortunately is not provided, and I cannot really find an good example of how to use this model to predict testing images. Is there a good example that I can use my trained model to predict multiple images in a batch way? Thank you for helping.

**Comment by michaelisard** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
